Task 1/main.css

```
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial;
+}
+
+body{
+  background-color: #DBDFE2;
+}
+
+/* TEXT */
+.txt{
+  font-size: 17px;
+  color: RGB(160, 174, 191);
+}
+
+.inptxt{
+  text-align: center;
+  margin: 10px;
+}
+
+
+.mainblock{
+  background-color: white;
+
+  border: 1px solid #CFD5DA;
+  border-radius: 5px;
+
+  width: 600px;
+  margin: auto;
+  padding: 30px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.inpfield{
+  background-color: RGB(237, 242, 247);
+
+  border: 2px solid white;
+  border-radius: 5px;
+
+  padding: 10px;
+
+  font-size: 15px;
+  color: RGB(74, 85, 103);
+}
+
+.inpfield:focus{
+  background-color: white;
+  outline: none;
+  border: 2px solid #9f7afb;
+}
+
+.input{
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: 10px;
+}
+
+.check{
+  width: 59%;
+  margin-left: 41%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.check > .txt{
```

`.txt`'s are useless, because you could simply use .check > h3. Different tags are the best helpers
```suggestion
.check > h3 {
```